oracle/datadog_checks/oracle/oracle.py

```diff
@@ -73,9 +75,10 @@ def check(self, instance):
         service_check_tags.extend(tags)
 
         with closing(self._get_connection(server, user, password, service, jdbc_driver, service_check_tags)) as con:
-            self._get_sys_metrics(con, tags)
-            self._get_process_metrics(con, tags)
-            self._get_tablespace_metrics(con, tags)
+            if only_custom_queries is None:
```

Comment:
```suggestion
            if not only_custom_queries:
```

Reply:
only_custom_queries wont be None because of https://github.com/DataDog/integrations-core/pull/5217/files#diff-582b365edd85d7b6c24fc67c09660d55R92